scripts/prep/glwu/exevs_glwu_wave_grid2obs_prep.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+###############################################################################
+# Name of Script: exevs_glwu_wave_grid2obs_prep.sh
+# Purpose of Script: To pre-process glwu forecast data into the same spatial
+#    and temporal scales as validation data.
+# Author: Samira Ardani (samira.ardani@noaa.gov)
+###############################################################################
+
+set -x
+
+##############################
+## grid2obs GLWU Model Prep 
+###############################
+
+cd $DATA
+echo "Starting grid2obs_prep for ${MODELNAME}_${RUN}"
+
+echo ' '
+echo ' *************************************'
+echo " *** ${MODELNAME}-${RUN} grid2obs prep ***"
+echo ' *************************************'
+echo ' '
+echo "Starting at : `date`"
+echo '-------------'
+echo ' '
+
+###############################################################################
+# Create today's GLWU individual fcst grib2 files and add them to the archive
+################################################################################
+
+mkdir -p ${DATA}/nc
+mkdir -p ${DATA}/gribs
+
+echo 'Copying GLWU wave grib2 and nc files'
+
+HHs='01 07 13 19'
+fcsts="$(seq -s ' ' 0 144)"
+
+for mtype in glwu glwu_lc grlc_2p5km grlc_2p5km_lc grlc_2p5km_lc_sr grlc_2p5km_sr grlr_500m grlr_500m_lc ; do
+	export mtype=${mtype}
+	if [ "${mtype}" == "grlc_2p5km" ] || [ "${mtype}" == "grlc_2p5km_lc" ]; then
+		for HH in ${HHs} ; do
+			filename="${MODELNAME}.${mtype}.t${HH}z.grib2"
+			COMINfilename="${COMINglwu}/${MODELNAME}.${INITDATE}/${filename}"
+			DATAfilename="${DATA}/gribs/${MODELNAME}.${mtype}.${INITDATE}.t${HH}z.grib2"
+			if [ ! -s ${COMINfilename} ]; then
+				echo echo "WARNING: No GLWU forecast was available for ${INITDATE}${HH}"
+				if [ SENDMAIL = YES ]; then
```

Comment:
Add `$` for SENDMAIL

Reply:
@malloryprow: I had added $ in the older commits.